### ISSUE TITLE
Integrate mjml-python?

### ISSUE THREAD

**Issue by blopker** [liminspace/django-mjml#184]
Hey I love to use this package in our project, but adding Node as a runtime dependency has me a bit scared. Are you interested in adding https://github.com/mgd020/mjml-python as an optional backend? It just wraps https://github.com/jdrouet/mrml (the Rust version of MJML) so no extra runtime dependency needed. Thanks!

**Comment by blopker** [comment]
For anyone who wants to try this now, I've copied the template tag from this library and copied it into my own project. There's not much to it when you don't have to talk to a separate process:

In `templatetags/mjm.py`:
```
from django import template
from mjml import mjml2html

register = template.Library()


class MJMLRenderNode(template.Node):
    def __init__(self, nodelist):
        self.nodelist = nodelist

    def render(self, context) -> str:
        mjml_source = self.nodelist.render(context)
        return mjml_render(mjml_source)


@register.tag
def mjml(parser, token) -> MJMLRenderNode:
    """
    Compile MJML template after render django template.

    Usage:
        {% mjml %}
            .. MJML template code ..
        {% endmjml %}
    """
    nodelist = parser.parse(("endmjml",))
    parser.delete_first_token()
    tokens = token.split_contents()
    if len(tokens) != 1:
        raise template.TemplateSyntaxError("'%r' tag doesn't receive any arguments." % tokens[0])
    return MJMLRenderNode(nodelist)


def mjml_render(mjml_source: str) -> str:
    return mjml2html(mjml_source)
```

**Comment by liminspace** [comment]
@blopker I think you can use `mjml-python` to create a HTTP or a TCP server and use a proper config to use that. It's better to use external service for rendering emails especially if you send a lot of emails.
But your idea is a good way to use mjml easier for smaller projects. I'm thinking about supporting customizable classes to make supporting alternative mjml-backends more flexible and allow developers use custom backends without making changes in `django-mjml`.

**Comment by aplex** [comment]
Tried @blopker version. Works well so far, and about 100x faster than cmd implementation of django-mjml. 
0.006 seconds per render vs 0.9 seconds

**Comment by blopker** [comment]
Nice! Glad it's working for you. We've been using it for a few months without issue. You can see our implementation here: https://github.com/totem-technologies/totem-server/blob/main/totem/email/emails.py

**Comment by Cruel** [comment]
Yeah, pretty much what I was wanting too @blopker. Even though it's simple, I went again and threw together a package for it for the people who may want it: https://pypi.org/project/django-mjml-template/
